iac-extensions/docker/src/main/java/org/sonar/iac/docker/parser/grammar/DockerGrammar.java

```diff
@@ -134,7 +137,7 @@ public ParamTree PARAM() {
         b.token(DockerLexicalGrammar.PARAM_PREFIX),
         b.token(DockerLexicalGrammar.PARAM_NAME),
         b.token(DockerLexicalGrammar.EQUALS_OPERATOR),
-        b.token(DockerLexicalGrammar.PARAM_VALUE)
+        b.optional(b.token(DockerLexicalGrammar.PARAM_VALUE))
```

Comment:
It's not a valid syntax, e.x. `COPY --option= src dest`, but in my opinion, it's better to have `--option=` as an `option` and not as `src` in `Tree`. Remember that files/directories may contain dash and equals character in name